README.md

```diff
@@ -3,9 +3,9 @@
 ```diff
 - WARNING! -
 
-The ROM this repository builds cannot be 'shifted', primarily due to some hardcoded pointers which have yet
-to be dumped. Thus this repository is currently in an experimental and research phase and cannot yet be used
-traditionally as a source code base for general changes.
+This repository is a work in progress, and while it can be used to make certain changes, it's still
```

Comment:
Id probably mention something about partial shiftability. IE any C file can be shifted but anything in ASM, Structs, and non-decompiled objects can not.

Reply:
This is probably a good idea yeah, we dont need to list specific examples but just let people be aware of some things that may not be shiftable

Reply:
I feel like it should be obvious that not everything can be done, and it will remain the case for a very long time, but I guess we can leave a small mention about it for now